src/test/scala/com/typesafe/npm/NpmSpec.scala

```diff
@@ -15,7 +15,7 @@ class NpmSpec extends Specification with NoTimeConversions {
   def withEngine[T](block: ActorRef => T): T = {
     val system = ActorSystem("test-system", classLoader = Some(this.getClass.getClassLoader))
     try {
-      val engine = system.actorOf(Trireme.props(), "engine")
+      val engine = system.actorOf(Node.props(), "engine")
```

Comment:
@mkurz this should be fixed now. The test was failing because it was a really old version of jsdom. The new version of jsdom no longer used contexify or node-gyp, so I switched the test to use aerospike so that the test would still invoke node-gyp